app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/MainActivity.java

```diff
@@ -77,17 +71,6 @@ private void switchFragment(Fragment f, int customAnimationIn, int customAnimati
     }
 
     @Override
-    public void onFragmentInteraction(int transition) {
-        switch (transition) {
-            case HOME_SCREEN_CLICK:
-                SpeechFragment f = SpeechFragment.newInstance();
-                switchFragment(f, FragmentTransaction.TRANSIT_NONE,
-                        R.anim.slide_out_up);
-                break;
-        }
-    }
-
-    @Override
```

Comment:
Oh wait, so actually the override in line 73 is that override because all that was deleted. So it looks like:
 @Override
public void speak...

now.